KEXPPower/Public API/AvailableStreams.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  AvailableStreams.swift
+//  KEXPPower
+//
+//  Created by Dustin Bergman on 2/27/22.
+//  Copyright © 2022 KEXP. All rights reserved.
+//
+
+import Foundation
+
+class AvailableStreams {
+    let livePlayback: [URL]
```

Comment:
Maybe this could be a dictionary versus array, with keys being the bitrate enum? Would be easier to maintain if we add more bitrates later

Reply:
Done